monitor.go

```diff
@@ -26,6 +26,7 @@ var dockerPassword = flag.String("password", "", "Registry password for pulling
 var registryHost = flag.String("registry-host", "", "Hostname of the registry being monitored")
 var repository = flag.String("repository", "", "Repository on the registry to pull and push")
 var baseLayer = flag.String("base-layer-id", "", "Docker V1 ID of the base layer in the repository")
+var testInterval = flag.String("run-test-every", "", "the time between test in minutes")
```

Comment:
Make this take in a generic duration string, and then you can use https://golang.org/pkg/time/#ParseDuration to parse it. That way, people can do `--run-test-every=20s` or `--run-test-every=2m`. Also, set the default string to "2m"